workers_controller.js

```diff
@@ -30,14 +28,8 @@ var PeersUpdateRules = require('./api/ws/workers/peers_update_rules');
 var Rules = require('./api/ws/workers/rules');
 var failureCodes = require('./api/ws/rpc/failure_codes');
 var Logger = require('./logger');
-
-var config = fs.readFileSync(
-	path.resolve(
-		process.cwd(),
-		`./config/${process.env.LISK_NETWORK}/config.json`
-	),
-	'utf8'
-);
+var AppConfig = require('./helpers/config.js');
+var config = AppConfig(require('./package.json'));
```

Comment:
Pass an extra param for `false` which will make AppConfig not to parse the command line params here.

Reply:
There are only three config values, that worker is dependent on configuration. 

```
config.consoleLogLevel
config.fileLogLevel
config.logFileName
```

If those can be handled by 

https://github.com/LiskHQ/lisk/blob/cb12a1e9d4d9d1bf65a7fc5d9427ee37e95d3084/app.js#L484-L491

that will leave no dependency  of worker over configuration file.

Reply:
I think change the config to send it to child process rather than reading it directly is not that easy since the childProcessOptions are sent async through IPC after starting the process.
In the current state, we need the config file at the initalization